test/jdk/javax/swing/JFileChooser/ZeroFileSizeCheck.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 1999, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright year is usually just the current year for new files. 
Was the test case created from a reproducer code?

Reply:
@

> The copyright year is usually just the current year for new files. Was the test case created from a reproducer code?

I have taken sample test case from the repo for reference and write mine test case, will update the copyright year.